utils/testman/cmd/differ/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func fatalPrint(logLine string) {
+	fmt.Println(logLine)
+	os.Exit(1)
+}
```

Comment:
nit: in general, it is better to `panic` because deferred functions are run when panicking and not run when using `os.Exit(1)`. Only mentioning as a nit here because there are no deferred functions here.

Reply:
Switched to panic 👍